css/style.css

```diff
@@ -1,80 +1,235 @@
+	content: '';
+	display: table;
+	clear: both;
 }
 
-/* Clear fix hack */
-.clearfix:after {
-     content: ".";
-     display: block;
-     clear: both;
-     visibility: hidden;
-     line-height: 0;
-     height: 0;
+/*Dropcaps*/
+
+p:first-letter {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 36px;
+    font-weight: 600;
+    float: left;
 }
 
-.clear {
-  clear: both;
+
+/*Container Sizing*/
+
+.container{
+
+	width:960px;
 }
 
-.alignright {
-  float: right;
-  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+main {
+	padding: 20px;
 }
 
-.alignleft {
-  float: left;
-  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+/*Top Nav Bar*/
+
+header{
+	background-color: #033048;
+	height:100px;
+	padding: 10px 20px;
+	width: auto;
+	margin-bottom: 40px;
 }
 
-/******************************************
-/* BASE STYLES
-/*******************************************/
+nav{
+	float: right;
+	padding: 0 0 10px 10px;	
+	width: auto;
+	font-size: 24px;
+	font-weight: 600;
+}
 
-body {
-     color: #000;
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
+header nav{
+	margin-top: 30px;
 }
 
+nav a {
+	color: yellow;
+	padding:20px;
+	text-decoration: none;
+	padding-top: 25px 45px 25px 25px;
+}
 
-/******************************************
-/* LAYOUT
-/*******************************************/
+h1{
 
-/* Center the container */
-#container {
-  width: 960px;
-  margin: auto;
+	color: white;
+	padding-top: 25px;
+	padding-left: 25px;
+	display: inline-block;
+	margin: 0;
+	font-size: 24px;
 }
 
-header {
+span{
 
+	font-size: 40px;
+	font-weight: 600;
 }
 
-footer {
+
+.article{
+	width: 700px;
 
 }
 
-/******************************************
-/* ADDITIONAL STYLES
-/*******************************************/
+.categories{
+	width: 240px;
+}
+
+.article, .categories{
+	float: left;
+	padding: 0 10px 10px 0;
+}
+
+
+img{
+
+	max-width: 100%;
+}
 
+h2{
+	font-size: 32px;
+	font-weight: 800;
+}
+
+.post {
+	font-size: 40px;
+	font-weight: 600;
+	display: block;
+	margin-bottom: 25px;
+}
 
+p{
+	font-size: 24px;
+	font-weight: 300;
+}
 
+.more{
+	display: block;
+	text-align: right;
+	font-size: 24px;
+	font-weight: 600;
+}
+
+.learn{
+	font-size: 22px;
+	font-weight: 600;
+}
+/*Sidebar*/
+
+aside{
+	margin-left: 15px;
+}
+
+ul a{
+	text-decoration: none;
+}
+
+.list{
+	list-style: none;
+	padding: 0;
+	text-decoration: none;
+}
+
+.about{
+	float: left;
+	margin-bottom: 30px;
+	font-size: 22px;
+	font-weight: 300;
+}
+
+aside ul li {
+	font-size: 24px;
+	font-weight: 300;
+}
+/*Footer*/
+
+.sign {
+	height:100px;
+	background-color: #f9e42e;
+	margin-bottom: -32px;
+	position: relative;
+}
+
+.button2 {
+	
+	background-color: #033048;
+	font-size: 14px;
+	border-radius: 5px;
+	color: white;
+	display: block;
+	width: 30%;
+	padding: 2px;
+	text-align: center;
+	line-height: 28px;
+	font-style: none;
+	position: absolute;
```

Comment:
Absolute positioning is tricky. I wouldn't recommend it for this particular case. 

Instead try something like this:

```css
.sign {
    background-color: #f9e42e;
    padding: 20px;
    text-align: center;
}

.button2 {
    background-color: #033048;
    font-size: 14px;
    border-radius: 5px;
    color: white;
    display: inline-block;
    text-align: center;
    line-height: 28px;
    font-style: none;
    text-decoration: none;
    font-weight: 800;
    padding: 5px 20px;
}
```